samples/quote/README.md

```diff
@@ -0,0 +1,27 @@
+# Remote Attestation Sample Code
+
+This directory contains sample source code to demonstrate how remote attestation works in ACON containers. More information on TDX remote attestation can be found at https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/TDX_Quoting_Library_API.pdf.
+
+This sample is comprised of a server and a client that communicate with each other over TCP. The server is an ACON container that generates TDX quotes upon requests from the client, which is a command line application running in the host.
+
+The client works with the server to retrieve a TD quote, along with the measurement logs and additional attestation data. It then verifies the quote, checks if RTMR values in the quote match those calculated from the measurement logs and if the server has attached the expected attestation data, and finally decodes/displays everything it receives . Addtionally, the quote is written to `quote.bin`, while RTMR logs and report data are written to `quote.json`.
+
+Simply type `make` to build both the server and the client. `docker`, `openssl`, and [`aconcli`](../../doc/GettingStarted.md#building-aconcli) are required in the build process.
+
+Running the sample requires a TDX enabled platform.
+
+- The server must be started first, by
+
+  ```sh
+  TCP_PORT=5555 ATD=1 ATD_KERNEL=/path/to/vmlinuz ATD_RD=/path/to/initrd.img aconcli run -ni -c:$TCP_PORT server.json

```

Comment:
Does it need a TCP port mapping between server and client?

Reply:
You are right! Which port?

Reply:
@xxu36 and @xiangquanliu, I added `ATD_TCPFWD=8080:8085`. Let me know if I'm wrong.

Reply:
I think that's right.

Reply:
yes, currently we are using this mapping